### ISSUE TITLE
Airbrake performance changes are filling logs with deprecation messages

### ISSUE THREAD

**Issue by wflanagan** [airbrake/airbrake-ruby#532]
In our logs since upgrading to 4.11.1 our logs are FULL (literally 2 lines per each controller load) of this: 
```
/root/.rbenv/versions/2.6.4/lib/ruby/gems/2.6.0/gems/airbrake-ruby-4.11.1/lib/airbrake-ruby.rb:369: warning: :start_time and :end_time are deprecated. Use :timing & :time instead
/root/.rbenv/versions/2.6.4/lib/ruby/gems/2.6.0/gems/airbrake-ruby-4.11.1/lib/airbrake-ruby.rb:451: warning: :start_time and :end_time are deprecated. Use :timing & :time instead
```

We do a lot of work to make sure our logs are clean and relevant. This deprecation warning is literally 66% of our logs at this point (as we use Lograge and each controller's log is 1 line).   And, it's not 1 time, it's literally on every request.

**Comment by bcm** [comment]
And it's costing me money, as my Papertrail log volume has gone through the roof.

**Comment by wflanagan** [comment]
Mine as well.  We keep running out, and then we have no logging for our app.

**Comment by kyrylo** [comment]
Sorry about this. I would recommend locking `airbrake-ruby` to the previous version for now, until I update the `airbrake` gem to the new API. I will ping you here once this is done & released.

**Comment by kyrylo** [comment]
I just released `airbrake v10.0.0`. Please upgrade to it and the warning will go away. Apologies again for the inconvenience 😔
